src/main/java/com/google/impactdashboard/servlets/ListProjectSummariesServlet.java

```diff
@@ -24,7 +24,9 @@
    */
   @Override
   public void init() {
-    projectInformationRetriever = ProjectInformationRetriever.create();
+    try { 
+      projectInformationRetriever = ProjectInformationRetriever.create();
+    } catch (java.io.IOException e) {}
```

Comment:
Ignoring exceptions is an anti-pattern. Please review Item 77: "Don't ignore exceptions" from Effective Java, as well as consider what the impact of this should be on the servlet (should it fail, should it retry, etc).

Reply:
Yes, sorry, this is really not a permanent solution. Once I implement the Credentials class, this error should be gone entirely, I only put in this try/catch block so that the code would compile while this IOException is still technically in the code and I could test what I have. Would you want me to throw a runtime exception there or something just so it's not ignored? This whole problem of there even being an error to catch here should be gone very soon (my next PR should get rid of it).

Reply:
ok, throwing a runtime exception for now is good, and then in the future removing entirely is fine. 

for the future, please annotate when something is temporary, otherwise the general assumption is that it is long term.

Reply:
Got it, will do!